internal/driver/driver_focus.go

```diff
@@ -77,19 +77,44 @@ func compileTagFilter(name, value string, ui plugin.UI, err error) (func(*profil
 	if value == "" || err != nil {
 		return nil, err
 	}
+
+	tagValuePair := strings.SplitN(value, "=", 2)
+	var key string
+	if len(tagValuePair) == 2 {
+		key = tagValuePair[0]
+		value = tagValuePair[1]
+	} else {
+		key = ""
+	}
+
 	if numFilter := parseTagFilterRange(value); numFilter != nil {
 		ui.PrintErr(name, ":Interpreted '", value, "' as range, not regexp")
-		return func(s *profile.Sample) bool {
-			for key, vals := range s.NumLabel {
-				for _, val := range vals {
-					if numFilter(val, key) {
+		labelFilter := func(vals []int64, key string) bool {
+			for _, val := range vals {
+				if numFilter(val, key) {
+					return true
+				}
+			}
+			return false
+		}
+		if key == "" {
+			return func(s *profile.Sample) bool {
+				for key, vals := range s.NumLabel {
+					if labelFilter(vals, key) {
 						return true
 					}
 				}
+				return false
+			}, nil
+		}
+		return func(s *profile.Sample) bool {
```

Comment:
I somewhat don't like the code duplication here with the function above, but I also don't see a good way to get rid of it...